dags/blocksec_plugin/tellor_oracle_operator.py

```diff
@@ -1,84 +1,30 @@
-from airflow.models.baseoperator import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from blocksec_plugin.web3_hook import Web3Hook
-from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
-import requests,json
+from blocksec_plugin.contract_interaction_operator import ContractInteractionOperator
+from blocksec_plugin.abis import SUBMIT_VALUE_ABI
```

Comment:
This is fine, this is the only tellor thing we do, no need to make a `TELLOR_ABI` this can be left as is